src/txkube/_network.py

```diff
@@ -113,10 +113,31 @@ def create(self, obj):
             d.addCallback(check_status, (CREATED,))
             d.addCallback(readBody)
             d.addCallback(loads)
-            def log_result(doc):
-                action.add_success_fields(response_object=doc)
-                return doc
-            d.addCallback(log_result)
+            d.addCallback(log_result, action)
+            d.addCallback(object_from_raw)
+            return d.addActionFinish()
+
+
+    def get(self, obj):
+        """
+        Issue a I{GET} to retrieve the given object.
+
+        The object must have identifying metadata such as a namespace and a
```

Comment:
Zoops!  Docs, indeed.  In fact, there's an explicit interface off to the side somewhere that I also forgot to update.  Thanks. :)